kotlinx-coroutines-test/api/kotlinx-coroutines-test.api

```diff
@@ -56,12 +56,11 @@ public final class kotlinx/coroutines/test/TestDispatchers {
 }
 
 public abstract interface class kotlinx/coroutines/test/UncaughtExceptionCaptor {
-	public abstract fun cleanupTestCoroutines ()V
+	public abstract fun cleanupTestCoroutinesCaptor ()V
 	public abstract fun getUncaughtExceptions ()Ljava/util/List;
 }
 
 public final class kotlinx/coroutines/test/UncompletedCoroutinesError : java/lang/AssertionError {
-	public fun <init> (Ljava/lang/String;Ljava/lang/Throwable;)V
-	public synthetic fun <init> (Ljava/lang/String;Ljava/lang/Throwable;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
```

Comment:
`AssertionError` in common code doesn't have a corresponding constructor.

Reply:
Sure. 

If it was the stable API, I would suggest providing constructor that invokes `initCause`, but here it is not necessary